### PR TITLE
week 39: correct the info about private namespaces

### PR DESCRIPTION
**Pull request by TomasTomecek**
@lachmanfrantisek @sakalosj this is what I had in mind - making sure that everyone in the public understands this work item has no impact on them

### CODE REVIEW

File: content/posts/September-2020.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ draft: false
 
 ## Week 39 (September 21st - September 25th)
 
-- Service supports private namespaces now. They need to be configured in the service config. [packit-service#831](https://github.com/packit/packit-service/pull/831)
+- Packit-service can be configured to work with private namespaces. This is plumbing work which we need right now for CentOS Stream. We are not planning to enable this for GitHub - packit-service will still work only for public repositories, private are ignored. [packit-service#831](https://github.com/packit/packit-service/pull/831)
```
Comment (csomh) [inline content/posts/September-2020.md:52]:
```suggestion
- Packit-service can be configured to work with private namespaces. This is plumbing work which we need right now for CentOS Stream. We are not planning to enable this for GitHub - packit-service will still work only for public repositories, private ones are ignored. [packit-service#831](https://github.com/packit/packit-service/pull/831)
```

**Review by csomh — APPROVED** [review]
Looks good, besides a small fix in wording.

**Comment by sakalosj** [comment]
> @lachmanfrantisek @sakalosj this is what I had in mind - making sure that everyone in the public understands this work item has no impact on them

thanks for fixing

**Review by lachmanfrantisek — COMMENTED** [review]
:+1:

**Pull request merged**